core/src/main/resources/error/error-classes.json

```diff
@@ -933,6 +933,11 @@
     ],
     "sqlState" : "42000"
   },
+  "TEMP_VIEW_NAME_CONTAINS_UNSUPPORTED_NAME_PARTS" : {
```

Comment:
error class name doesn't have to be super clear to make it short. How about `TEMP_VIEW_NAME_TOO_MANY_NAME_PARTS `

Reply:
I see. Updated